modules/os2forms_forloeb/src/Controller/MaestroNotificationController.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Drupal\os2forms_forloeb\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Url;
+use Drupal\os2forms_digital_post\Model\Document;
+use Drupal\os2forms_forloeb\MaestroHelper;
+use Drupal\webform\WebformInterface;
+use Drupal\webform\WebformSubmissionInterface;
+use Drupal\webform\WebformSubmissionStorageInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Maestro notification controller.
+ */
+class MaestroNotificationController extends ControllerBase {
+
+  /**
+   * Constructor.
+   */
+  public function __construct(
+    private readonly WebformSubmissionStorageInterface $webformSubmissionStorage,
+    private readonly MaestroHelper $maestroHelper
+  ) {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('entity_type.manager')->getStorage('webform_submission'),
+      $container->get(MaestroHelper::class)
+    );
+  }
+
+  /**
+   * Preview action.
+   */
```

Comment:
full doc missing

Reply:
I'm not quite sure what you mean with “full doc missing”, but I've cleaned up some comment and updated the PhpDoc blocks with all parameters.

Our coding standards do not enforce that we document (all) parameters and therefore I hadn't done it before. In my opinion comments like

```php
   * @param string $handler
   *   The handler ID.
   * @param string $notification_type
   *   The notification type.
```

adds absolutely no value.